.dependencies_checksum

```diff
@@ -0,0 +1 @@
+e9230c2f0c6db0ae559ee7be5d4045cd6c2af6557e3259d7c8bb67649a9be760  -
```

Comment:
Is there meant to be a `  -` at the end?

Reply:
Yeah, that's the entire output of `sha256sum` - doesn't seem like a big deal

Reply:
Oh the `-` is actually the name of the file being checksummed.

Since the input is from the shell (redirected from `cat`), it displays `-`.